src/App.js

```diff
@@ -1,5 +1,23 @@
-function App() {
-  return <div className="App">Hello world</div>
+import WorkoutCard from './components/WorkoutCard'
+import React from 'react'
+import styled from 'styled-components/macro'
+import { results } from './staticWorkoutList.json'
+
+export default function App() {
+  return (
+    <AppLayout>
+      <h2> Select your workout:</h2>
+      {results.map(results => (
+        <WorkoutCard key={results.name} {...results} />
```

Comment:
Es ist etwas verwirrend, dass du in `map()` ebenfalls `results`(Mehrzahl) verwendest. Da würde ich vielleicht nur `result` nehmen.

Was macht `{...results}` bei WorkoutCard? Du weißt das keiner property zu.